bool enabled_status
bool battery_unsafe
float32 loop_freq